include/metal/list/sort.hpp

```diff
@@ -3,13 +3,15 @@
 
 #include "../config.hpp"
 #include "../detail/sfinae.hpp"
+#include "../lambda/lambda.hpp"
 #include "../list/list.hpp"
 #include "../number/if.hpp"
+#include "../number/less.hpp"
 
 namespace metal {
     /// \cond
     namespace detail {
-        template<class lbd>
+        template<class lbd = metal::lambda<metal::less>>
```

Comment:
This default argument is not necessary, since by the time this is instantiated all arguments are known.
Removing it may help MSVC.

Reply:
I am not sure to understand your comment. MSVC emits an ICE for the code in #107. There, the only defaulted parameter is the `lbd` of the alias template.

Reply:
Never mind, my bad, I didn't realize it was already defined this way on that PR.